### PR TITLE
feat(design): update control interface

### PR DESCRIPTION
**Pull request by JianKangEgon**
## Description

The previous description of control interface including diagram, input & output summary, etc. could be complemented. 
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/design/autoware-interfaces/components/control.md [hunk 1]
```diff
@@ -2,57 +2,67 @@
 
 ![Node diagram](images/Control-Bus-ODD-Architecture.drawio.svg)
 
-## Inputs
+## Overview
 
-### Vehicle kinematic state
+Control stack plays a role in transferring the decision from "brain" to "hand / foot". From the previous planning component, the autonomous vehicle acquires the task trajectory. So the following module, i.e. Control stack is supposed to focus on how to maneuver the vehicle to follow the received trajectory.
```
Comment (xmfcx) [inline docs/design/autoware-interfaces/components/control.md:7]:
```suggestion
Control component takes in a trajectory to follow from the planning component. Then it generates control signals such as target velocity, acceleration and steering angle etc. to follow the trajectory.
```

I think it's better to keep it simple like this, what do you think?

**Comment by xmfcx** [comment]
Oh sorry, I think you've meant to edit 
- https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/design/autoware-architecture/control/index.md
  - This is for internal control component design.
- **NOT** https://github.com/autowarefoundation/autoware-documentation/blob/main/docs/design/autoware-interfaces/components/control.md
  - This is for how Autoware should interface with outside.

@JianKangEgon Can you confirm and edit the correct file?

**Comment by JianKangEgon** [comment]
> 

hi Fatih, thanks for your reply. Actually, this pr was proposed for internal component at the very beginning. However, Mitsudome-san thought it's more like interface rather than component. So, I'm a little confused. @mitsudome-r  Could you have a look at it?

**Comment by JianKangEgon** [comment]
@mitsudome-r  hi, there is not any feedback recently regarding this pr. Since Fatih @xmfcx considered this as control component, so if there is not any conclusion, I'll close this pr in one week and propose a new one if needed.

**Pull request closed without merging**